Processing_Sample/CProcessing/Source/vect.h

```diff
@@ -37,8 +37,8 @@ unsigned int vect_ins(vect_t *v, unsigned int pos);
 #define VECT_GENERATE_TYPE(TYPE) VECT_GENERATE_NAME(TYPE, TYPE)
 #define VECT_GENERATE_NAME(TYPE, NAME)											\
 typedef struct {																\
-	int size;																	\
-	int capacity;																\
+	size_t size;																	\
```

Comment:
You also need to change the vect_t struct within the vect.c file in order to reflect the size changes. As it is now the program will not run because of uint -> int casting, causing a read access violation.